### PR TITLE
Revert "Revert "Navbar.js unordered list for links""

### PR DESCRIPTION
**Pull request by evanmorristest**
This reverts commit b906b0e24d7a623916e1df5813c067a7745599e2.

### CODE REVIEW

File: src/components/navbar.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```
Comment (nmunro) [inline src/components/navbar.js]: Is this needed?

**Review by nmunro — DISMISSED** [review]

**Review by ap42m — DISMISSED** [review]
i think we're using it as a placeholder :)

File: src/components/navbar.js (hunk 1 above)
Comment (evanmorristest) [reply src/components/navbar.js]: good point, will fix this now

Comment (ap42m) [reply src/components/navbar.js]: we dont need React for this component becuase it is not being used

**Review by ap42m — COMMENTED** [review]
sweet!

**Review by nmunro — APPROVED** [review]
Awesome!

**Review by ap42m — APPROVED** [review]

**Pull request merged**